pkg/apis/build/v1alpha1/clusterbuildstrategy_types.go

```diff
@@ -5,9 +5,19 @@
 package v1alpha1
 
 import (
+	"strconv"
+
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	// LabelClusterBuildStrategyName is a label key for defining the build strategy name
```

Comment:
It should be the `cluster` build strategy name

Reply:
Again, we do not have a clear consistency or symmetry in the naming. What is called the "build strategy"  is at some point called "namespace build strategy". So with other words, the term "build strategy" is a bit overloaded in my opinion. I do not mind changing the comment there a bit, but at the end of the day, both strategies are technically speaking build strategies.

Reply:
Yes, but in current logic, I think `build strategy` may make people confusing